assets/scss/_custom.scss

```diff
@@ -349,52 +349,62 @@ footer {
 }
 
 main {
-  .td-content table code,
-  .td-content>table td {
-    word-break: break-word;
-  }
 
-/* SCSS Related to the Metrics Table */
+/* SCSS Related to the Metrics list */
+
+  div.metric:nth-of-type(odd) { // Look & Feel , Aesthetics
+    background-color: $light-grey;
+  }
 
-    @media (max-width: 767px) { // for mobile devices, Display the names, Stability levels & types
```

Comment:
Is this breaking the mobile view?

Reply:
This was necessary when we had the old tabular format @tengqm, Now It just looks fine without it